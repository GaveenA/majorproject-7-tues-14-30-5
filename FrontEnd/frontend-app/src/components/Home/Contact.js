import React from 'react'

function Contact() {
    return (
        <div>
            <h1>CONTACT PAGE HERE</h1>
        </div>
    )
}

export default Contact