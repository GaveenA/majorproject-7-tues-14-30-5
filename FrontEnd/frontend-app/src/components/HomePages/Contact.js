import React, {Component} from 'react'

class Contact extends Component {
    render() {
        return (
            <h1>CONTACT PAGE HERE</h1>
        )
    }
}

export default Contact