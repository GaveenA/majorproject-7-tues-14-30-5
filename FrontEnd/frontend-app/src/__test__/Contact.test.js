import React from 'react';
import ReactDOM from 'react-dom';
import {render} from '@testing-library/react';
import Contact from '../components/Home/Contact.js';
import renderer from 'react-test-renderer';



it("renders Contact button without crashing",() => {
    const dev = document.createElement("dev");
    ReactDOM.render(<Contact></Contact>, dev);
})


it("Contact Class Snapshot is Matched",() => {
 
    const tree = renderer.create(<Contact></Contact>).toJSON();
    expect(tree).toMatchSnapshot();
})