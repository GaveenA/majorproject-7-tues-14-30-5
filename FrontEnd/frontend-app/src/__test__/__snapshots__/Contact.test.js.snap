// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Class Snapshot is Matched 1`] = `
<div>
  <h1>
    CONTACT PAGE HERE
  </h1>
</div>
`;
